packages/app/src/debug/DebugPageHeader.cy.tsx

```diff
@@ -0,0 +1,54 @@
+import { DebugPageFragmentDoc } from '../generated/graphql-test'
+import DebugPageHeader from './DebugPageHeader.vue'
+
+const generateTags = (num): any => new Array(num).fill(null).map((_, i) => ({ id: `${i}`, name: `tag${i}`, __typename: 'CloudRunTag' }))
```

Comment:
```suggestion
const generateTags = (num) => new Array(num).fill(null).map((_, i) => ({ id: `${i}`, name: `tag${i}`, __typename: 'CloudRunTag' }))
```

If you remove `any` TS can infer the usage, if you hold you mouse over `generateTags` it should infer the return type as an object w/ the defined properties.

Reply:
I just realized that we do not need to generate tags for this component.